components/widgets/TableData.js

```diff
@@ -177,40 +184,67 @@ EnhancedTableToolbar.propTypes = {
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    width: '100%',
+    width: "100%",
   },
   paper: {
-    width: '100%',
+    width: "100%",
     marginBottom: theme.spacing(2),
   },
   table: {
     minWidth: 750,
   },
   visuallyHidden: {
     border: 0,
-    clip: 'rect(0 0 0 0)',
+    clip: "rect(0 0 0 0)",
     height: 1,
     margin: -1,
-    overflow: 'hidden',
+    overflow: "hidden",
     padding: 0,
-    position: 'absolute',
+    position: "absolute",
     top: 20,
     width: 1,
   },
 }));
 
 export default function EnhancedTable() {
   const classes = useStyles();
-  const [order, setOrder] = React.useState('asc');
-  const [orderBy, setOrderBy] = React.useState('calories');
+  const [order, setOrder] = React.useState("asc");
+  const [orderBy, setOrderBy] = React.useState("killing");
   const [selected, setSelected] = React.useState([]);
   const [page, setPage] = React.useState(0);
   const [dense, setDense] = React.useState(false);
   const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const [rows, setRows] = React.useState([]);
+
+  React.useEffect(() => {
+    async function fetchAPI() {
```

Comment:
This totally works and I’m happy to approve. Just want to point out that we also fetch data in a method called `getServerSideProps` or something like that in the `index.js` file. That has the benefit/behavior of fetching the data before it gets to the browser. Not sure which we rather do long term, but we probably will want to standardize the approach eventually.

Reply:
Will approve and merge this @tam-nguyen . I added this to our backlog to improve on later -> https://trello.com/c/fbQzeGpB/66-update-code-for-tabular-data-table-so-that-it-pulls-data-using-getserversideprops-instead-of-fetchapi

Also, congrats on your first commit!!!